From f6c069cfb438086206ef34206e322e8dab18973a Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 20 Jan 2017 14:59:59 -0800
Subject: [PATCH] IMGAPI-612 IMGAPI suffering from IMGAPI-599 which breaks
 http_proxy support Reviewed by: Julien Gilli <julien.gilli@joyent.com>
 Approved by: Trent Mick <trentm@gmail.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 6670eed..7596fb1 100644
--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "progbar": "git+https://github.com/trentm/node-progbar.git#a4c56e6",
     "restify": "4.1.1",
     "rimraf": "2.2.6",
-    "sdc-clients": "10.0.1",
+    "sdc-clients": "10.0.4",
     "semver": "3.0.1",
     "sshpk": "1.10.0",
     "trace-event": "1.2.0",
-- 
2.21.0

